carbon-design-system/carbon
# test(ComboBox): add vrt

Closes #9268 

Adds percy snapshots for ComboBox

#### Testing / Reviewing

- view Percy test details, check ComboBox snapshot, ensure styles are applied correctly, approve snapshot if visuals are correct :) 
- ensure all tests are passing
